ibis/backends/flink/tests/conftest.py

```diff
@@ -0,0 +1,13 @@
+from pathlib import Path
+
+import ibis
+from ibis.backends.tests.base import BackendTest, RoundAwayFromZero, UnorderedComparator
+
+FLINK_CATALOG = "default_catalog"
+FLINK_DATABASE = "default_database"
+
+
+class TestConf(UnorderedComparator, BackendTest, RoundAwayFromZero):
+    @staticmethod
+    def connect(data_directory: Path):
+        return ibis.flink.connect(catalog=FLINK_CATALOG, database=FLINK_DATABASE)
```

Comment:
Since we're not implementing execution tests yet, I don't think you should define this subclass at all. This whole file may be deleted. Deleting this should also let you revert the changes in `ibis/backends/conftest.py`.